dts/bindings/sensor/bosch,bmi270-spi.yaml

```diff
@@ -0,0 +1,10 @@
+# Copyright (c) 2022, Nordic Semiconductor ASA
+# SPDX-License-Identifier: Apache-2.0
+
+description: |
+    The BMI270 is an inertial measurment unit. See more info at:
```

Comment:
You can define this once in bosch,bmi270.yaml and not copy/paste it into the spi and i2c files. Please fix the "measurement" typo.